### DISCUSSION TITLE
Residuals

### DISCUSSION THREAD

**Discussion by vcforsubaffairs** [Boulder-Investment-Technologies/lppls#82] [Q&A]
Hi,

Could you please provide some solution for extracting the difference between the ln of my data and lppls fit? 

Best regards,
Alek

**Comment by Joshwani** [comment]
Hi Alek, Yes I think I should be able to add this in. Standby for PR.
Best,
Josh

**Reply by Joshwani** [reply]
@vcforsubaffairs do you want residuals for every fit in the conf score calc or just for single fit?

**Reply by Joshwani** [reply]
https://github.com/Boulder-Investment-Technologies/lppls/pull/84
